src/js/navButtons.js

```diff
@@ -0,0 +1,115 @@
+                my: "left+35 bottom-20"
+            }
+        },
+        selectors: {
+            back: ".gpiic-firstDiscovery-navButtons-back",
+            next: ".gpiic-firstDiscovery-navButtons-next"
+        },
+        strings: {
+            back: "Back",
+            next: "Next",
+            start: "Let's start",
+            finish: "Finish"
+        },
+        modelListeners: {
+            currentPanelNum: "{that}.setButtonStates"
+        },
+        listeners: {
+            "onCreate.bindBack": {
+                "this": "{that}.dom.back",
+                "method": "click",
+                args: ["{that}.backButtonClicked"]
+            },
+            "onCreate.bindNext": {
+                "this": "{that}.dom.next",
+                "method": "click",
+                args: ["{that}.nextButtonClicked"]
+            }
+        },
+        invokers: {
+            setButtonStates: {
+                funcName: "gpii.firstDiscovery.navButtons.setButtonStates",
+                // Calls on "{that}.backTooltip", "{that}.nextTooltip" to force the instantiate of these sub-components
+                args: ["{that}", "{that}.backTooltip", "{that}.nextTooltip"]
+            },
+            setModel: {
+                funcName: "gpii.firstDiscovery.navButtons.setModel",
+                args: ["{that}", "{arguments}.0"]
+            },
+            backButtonClicked: {
+                funcName: "gpii.firstDiscovery.navButtons.setModel",
+                args: ["{that}", -1]
+            },
+            nextButtonClicked: {
+                funcName: "gpii.firstDiscovery.navButtons.setModel",
+                args: ["{that}", 1]
+            }
+        },
+        components: {
+            backTooltip: {
+                type: "fluid.tooltip",
+                container: "{that}.dom.back",
+                options: "{navButtons}.options.tooltipOptions"
+            },
+            nextTooltip: {
+                type: "fluid.tooltip",
+                container: "{that}.dom.next",
+                options: "{navButtons}.options.tooltipOptions"
+            }
+        }
+    });
+
+    gpii.firstDiscovery.navButtons.setButtonStates = function (that, backTooltip, nextTooltip) {
+        var currentPanelNum = that.model.currentPanelNum,
+            strings = that.options.strings,
+            backButton = that.locate("back"),
+            nextButton = that.locate("next"),
+            isFirstPanel = currentPanelNum === that.options.panelStartNum,
+            nextLabel = isFirstPanel ? strings.start : (currentPanelNum === that.options.panelTotalNum ? strings.finish : strings.next);
+
+        backButton.prop("disabled", isFirstPanel);
+        backButton.toggle(!isFirstPanel);
+        backButton.html(strings.back);
+        nextButton.html(nextLabel);
+        backTooltip.updateContent(strings.back);
+        nextTooltip.updateContent(nextLabel);
+    };
+
+    gpii.firstDiscovery.navButtons.setModel = function (that, toChange) {
```

Comment:
This method might need to be renamed. I feel like the name implies that you could actually set the model to a particular value, but it's really just an offset from it's current value, which you may or may not know.